gradle/dependencies.gradle

```diff
@@ -13,7 +13,7 @@ ext {
       mapboxSdkDirectionsModels : '5.1.0-beta.1',
       mapboxEvents              : '4.5.1',
       mapboxCore                : '1.3.0',
-      mapboxNavigator           : 'sf-keypoints-SNAPSHOT-8',
+      mapboxNavigator           : 'master-SNAPSHOT-0',
```

Comment:
is this the most up-to-date snapshot @mskurydin @Guardiola31337?

Reply:
Actually not, `ms-bearing-from-shape-on-interpolation-SNAPSHOT-2` is the most up to date. Will bump.